### PR TITLE
Adds Hydrogen - A new gas to play with!

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

Adds hydrogen - Does everything that tritium does but without the rads when it burns. Also gets converted to tritium when exposed to radiation like pluoxium does.

Also tweaks tritium (and thus hydrogen) to have a specific heat of 40. Real life hydrogen and its isotopes have ten times the heat capacity of oxygen, so this one is only fair.

Currently lacking in special features, but I'll be adding that when I add the electrolysis chamber to convert water vapor into hydrogen and Oxygen, and when I add deuterium and the neutron moderation chamber to enrich isotopes of hydrogen into deuterium and eventually tritium.

The equilibrium reaction was scrapped since it would cause floating point bugs in edge cases (I ran all my top sekrit atmos techniques on it), I'll add it in a future revision when all 3 isotopes are available for use.

I'll be adding support for fusion in the future as well. Maybe we can finally move on to a version of fusion that isn't super simplistic to execute, with the only hard part being the tritium harvesting and heating setup.

Also if this forces a merge conflict with my other PR, that's because git for some reason counts a comma change (dunno why) and the re-removal of stimball. Pretty sure that'll be easy to solve.
#### Changelog

🆑 
rscadd: Adds hydrogen. Has all the properties of tritium except it absorbs radiation to morph into tritium and doesn't emit any when it burns.
tweak: Modifies tritium's heat capacity to 40 
/:cl:

### CODE REVIEW

**Comment by Mr-Space** [comment]
travis god is not satisfied

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -389,7 +452,7 @@
 		/datum/gas/tritium = 30,
 		/datum/gas/plasma = 10,
 		/datum/gas/bz = 20,
-		/datum/gas/nitryl = 30,
```
Comment (SomeguyManperson) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: Put that comma back and travis should be fine

Comment (ghost) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: fixed

**Comment by CMOisLing** [comment]
It should be invisible and non toxic to breath. So it could be spread without alerting anyone kinda like real hydrogen.

**Comment by MegaEmpirical** [comment]
Chemical

**Comment by MegaEmpirical** [comment]
I doubt it would actually be chemically similar to trit.

Description summary is lacking.

**Comment by ghost** [comment]
> I doubt it would actually be chemically similar to trit.
> 
> Description summary is lacking.

Literally the biggest problem with isotope separation that nazi germany faced with producing enough trit for their atomic weapons project was the fact that all isotopes have similar chemical properties and required special physical separation procedures. Since they basically do the exact same thing, this is a copy, the only special bits will be fusion when I add it in.

**Comment by ghost** [comment]
> It should be invisible and non toxic to breath. So it could be spread without alerting anyone kinda like real hydrogen.

That's how it currently behaves 😈 

At least until we get a sprite for it.

**Comment by MegaEmpirical** [comment]
> > I doubt it would actually be chemically similar to trit.
> > Description summary is lacking.
> 
> Literally the biggest problem with isotope separation that nazi germany faced with producing enough trit for their atomic weapons project was the fact that all isotopes have similar chemical properties and required special physical separation procedures. Since they basically do the exact same thing, this is a copy, the only special bits will be fusion when I add it in.

Disregard what I said, I am a dumbfuck who forgot about H3 and was just thinking of trit as a fake gas like plasma.

**Comment by DerelictDrone** [comment]
Hold on a sec, what about the chem hydrogen? Would splashing some release hydrogen as a gas like liquid oxygen and (unstabilized)plasma do?

**Comment by Moja10** [comment]
Add the ability to make water vapor from hydrogen and oxygen!!!!!!!!!

**Comment by ghost** [comment]
> Add the ability to make water vapor from hydrogen and oxygen!!!!!!!!!

Yep it's there already.

Also, I was planning to let this sit and be merged before I started working on additional improvements. Should I just commit and push those or wait?

**Comment by Jarod1200** [comment]
I cant wait to make Hindenburg station 13 with all the hydrogen from tritium production water vapor

**Comment by ghost** [comment]
Yeah this is never gonna get merged. I'll go back to this someday, eventually.

**Pull request closed without merging**